DOC-7920 --  Provide console Logging & basic troubleshooting tips

ibsoln opened the pull request:
https://issues.couchbase.com/browse/DOC-7920

bbrks commented on modules/ROOT/pages/advance/logging.adoc:
```diff
@@ -125,30 +189,106 @@ The default log level is `info`.
+
+You can define the required {xref-sgw-bmk-config-properties-log-console-key} within your configuration file and-or use the Admin REST API (see: <<eg-setloggingwithapi>>).
+
+.List of Available Log Keys
+[#tbl-logkeylist,cols="^2,8"]
+|===
+h|Log Key
+h|Description
```
Quite a few of these log keys no longer exist (The Shadow log key for example):

Here's the full list of log keys in use in the current version:

https://github.com/couchbase/sync_gateway/blob/4b2567e1968973eab0330d07d06035cecc15848e/base/log_keys.go#L54-L75 

Let me know if you need any descriptions for more of them:
- `SGCluster` is related to the sharded import and HA sg-replicate

bbrks commented on modules/ROOT/pages/advance/logging.adoc:
```diff
@@ -71,34 +84,85 @@ You configure your logging requirements in the `sync-gateway-config.json` file,
 
-The console output of Sync Gateway can be filtered down via <<log-levels>> and <<log-keys>>, and you can tweak this as much as you like without impacting Support's ability to analyze the log files.
+[#lbl-log-api]
+=== Admin REST APi
```
```suggestion
=== Admin REST API
```

bbrks reviewed (changes requested):
Couple of small comments

ibsoln committed:
DOC-7920 --  Provide console Logging & basic troubleshooting tips

https://issues.couchbase.com/browse/DOC-7920

Other commits subsumed in here are:

Update the available keys table per feedback

Correct APi to API

Co-authored-by: Ben Brooks <ben@bbrks.me>

Update logging.adoc

DOC-7920 -- Enhance logging and troubleshooting  content

https://issues.couchbase.com/browse/DOC-7920

Provide console Logging & basic troubleshooting tips
